File: src/main/java/pl/exsio/querydsl/entityql/examples/spring_data_jdbc/entity/Order.java
package pl.exsio.querydsl.entityql.examples.spring_data_jdbc.entity;

import org.springframework.data.annotation.Id;

import javax.annotation.Nonnull;
import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class Order implements Serializable {

    @Id
    private final Long id;

    private final User user;

    private List<OrderItem> items = new ArrayList<>();

    public Order(Long id, User user) {
        this.id = id;
        this.user = user;
    }


}

File: kotlin-gradle-spring-data-jdbc-static/src/main/kotlin/pl/exsio/querydsl/entityql/examples/spring_data_jdbc/entity/Order.kt
package pl.exsio.querydsl.entityql.examples.spring_data_jdbc.entity

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Column

class Order(@Id @Column("ORDER_ID") var id: Long, var user: User<*>, var items: List<OrderItem>)
